### ISSUE TITLE
Time for a new release?

### ISSUE THREAD

**Issue by rossburton** [cryptodev-linux/cryptodev-linux#93]
There's not been a lot of commits since the last release, but they're pretty important ones:

- [zero copy: Fix build for Linux 6.4](https://github.com/cryptodev-linux/cryptodev-linux/commit/592017c3a910a3905b1925aee88c4674e9a596b7)
- [cryptodev_verbosity: Fix build for Linux 6.4](https://github.com/cryptodev-linux/cryptodev-linux/commit/99ae2a39ddc3f89c66d9f09783b591c0f2dbf2e9)
- [Move recent Linux version #ifdefs from v6.4 to v6.5](https://github.com/cryptodev-linux/cryptodev-linux/commit/bb8bc7cf60d2c0b097c8b3b0e807f805b577a53f)
- [Fix build for Linux 6.7-rc1](https://github.com/cryptodev-linux/cryptodev-linux/commit/5e7121e45ff283d30097da381fd7e97c4bb61364)
- [Fix build for linux 5.10.220](https://github.com/cryptodev-linux/cryptodev-linux/commit/157a624d7a892ea5fb2df4bbd4e71c008adbecb7)

Basically using 6.4+ means needing to build this from git, not release.

**Comment by cristian-stoica** [comment]
Yes - I think we can mark the release by the end of this week if there are no other patches pending on people's machines

**Comment by rossburton** [comment]
We do have one patch that could be upstreamed.  Basically because we build the headers and the kernel module in separate packages we'd like `make install` to be able to be split up. Something like this:

```
install: modules_install headers_install

modules_install:
	$(MAKE) $(KERNEL_MAKE_OPTS) modules_install

headers_install:
	install -m 644 -D crypto/cryptodev.h $(DESTDIR)/$(includedir)/crypto/cryptodev.h
```

This would let us install the pieces we want, instead of having to patch the Makefile.

**Comment by rossburton** [comment]
Also, removing the `-Werror` in `tests/Makefile` would be good: makefiles should never force errors on as you don't control the build environment (https://github.com/cryptodev-linux/cryptodev-linux/issues/67).

**Comment by cristian-stoica** [comment]
It makes sense. Please submit the two patches to include them
